packages/server-wallet/src/models/channel.ts

```diff
@@ -173,6 +177,67 @@ export class Channel extends Model implements RequiredColumns {
     });
   }
 
+  public static directFundingStatus(
+    supported: SignedStateVarsWithHash | undefined,
+    fundingFn: (address: Address) => ChannelStateFunding,
+    myParticipant: Participant,
+    fundingStrategy: FundingStrategy
+  ): FundingStatus {
+    if (fundingStrategy !== 'Direct') {
+      return 'Uncategorized';
```

Comment:
Isn't it the case that the direct funding _status_ is independent of the funding _strategy_?

It is unlikely, but possible, that
1. Alice & Bob have a directly funded ledger L, funding a channel C
2. Alice goes on vacation
3. Bob challenges in L, and transfers funds to C

Now, C is directly funded, though its funding strategy was `'Ledger'`.

Reply:
Should the `fundingStrategy` get updated to `Direct` in the scenario you describe?